drones:
  - count: 4
  - names: ["drone1", "drone2", "drone3", "drone4"]
  - drone1: 
    - x: -4
      y: -2
      z: 3

    - x: -6
      y: 0
      z: 3

    - x: -4
      y: 2
      z: 3

    - x: -2
      y: 0
      z: 3

  - drone2:
    - x: 4
      y: -2
      z: 3.5

    - x: 2
      y: 0
      z: 3.5

    - x: 4
      y: 2
      z: 3.5

    - x: 6
      y: 0
      z: 3.5
      
  - drone3:
    - x: 0
      y: 2
      z: 4

    - x: -2
      y: 4
      z: 4

    - x: 0
      y: 6
      z: 4

    - x: 2
      y: 4
      z: 4

  - drone4:
    - x: 0
      y: -6
      z: 4.5

    - x: -2
      y: -4
      z: 4.5

    - x: 0
      y: -2
      z: 4.5

    - x: 2
      y: -4
      z: 4.5
    
    
uas_numbers:
  - 6
  - 8
  - 9
  - 10